modal/cls.py

```diff
@@ -225,6 +225,14 @@ def __getattr__(self, k):
 
 
 class _Cls(_Object, type_prefix="cs"):
+    """
+    Cls adds method pooling and [lifecycle hook](/docs/guide/lifecycle-functions) behavior
```

Comment:
I tried to find documentation on the "method pooling" behavior and could only find it in the changelog? That seems like a documentation gap that needs filling.

Reply:
This is missing "parameterization" BTW. Agreed that we should document that method pooling behavior somewhere. Somewhat unclear to me if it is in itself a "feature", or just a fact about how classes work.

BTW maybe a pet peeve (depending on whether you consider [PEP257](https://peps.python.org/pep-0257/#multi-line-docstrings) a standard) but I think the one-line summary in a docstring really should fit on one line.

It's unfortunate that we need to spell out the cross-ref to the API docs with a url path, which is most of why your summary spills over. In sphinx you could write it as
```
:class:`Function`
```
and the docs framework would handle the hyperlink in the HTML build.